src/rules/__tests__/no-if.test.ts

```diff
@@ -552,30 +552,14 @@ ruleTester.run('if statements', rule, {
     },
   ],
   invalid: [
-    {
```

Comment:
why is this deleted?

Reply:
Because it's a switch test in the `if` test block; switches are tested in [this group](https://github.com/jest-community/eslint-plugin-jest/pull/587/files#diff-af685d385381b07d87cda847c6b0ae4dR71).

I must have done a bad copy and paste; this test fails because it's data property has it expecting `if` instead of `switch` for `condition`.

Reply:
[Here is it's duplicate](https://github.com/jest-community/eslint-plugin-jest/pull/587/files#diff-af685d385381b07d87cda847c6b0ae4dR179-R193) 🙂

Reply:
👍